tests/test_contrib/test_scatter.py

```diff
@@ -147,6 +147,17 @@ def test_integrated_scatter(self):
         visualizer = ScatterViz(features=features)
         visualizer.fit_transform_poof(X[:, :2], y)
 
+    def test_alpha_param(self):
+        """
+        Test that the user can supply an alpha param on instantiation
+        """
+        # Instantiate a scatter plot and provide a custom alpha
+        visualizer = ScatterVisualizer(
+            x="temperature", y="light", alpha=0.7
+        )
+
+        assert visualizer.alpha == 0.7
```

Comment:
Awesome - I really appreciate it when PRs also include tests, it makes the review process tons easier and ensures we don't break things in the future!  

This test is sufficient for this PR, but because you mentioned you'd be interested in updating other visualizers with alpha parameters, I was wondering if you'd like to explore our [image similarity testing](http://www.scikit-yb.org/en/latest/contributing.html#image-comparison-tests)? This adds a bit more complexity to the development process but might be worth figuring out if you're interested in working on Yellowbrick more routinely. 

Alternatively for the other PRs, you could do something a bit simpler than image similarity, by mocking the `viz.ax.scatter` method and asserting that it is called with the supplied alpha. Let me know if you'd like me to go into more detail on this or an image similarity, I'd be happy to help!

Reply:
Thanks! I'm also new to testing. This is a great learning experience. I'm maybe not quite ready for the visual test, but I'll take a look at the link you sent!

Reply:
Ok, no worries at all! The image similarity tests aren't necessarily difficult but do require some extra process to save baseline images. When we merge this in, I can quickly write a mock test as an example of what mocking might look like, in case you'd like to use that for the other PRs.